openhands/llm/llm.py

```diff
@@ -53,6 +53,14 @@
     'claude-3-opus-20240229',
 ]
 
+# function calling supporting models
+FUNCTION_CALLING_SUPPORTED_MODELS = [
+    'claude-3-5-sonnet-20240620',
+    'claude-3-5-sonnet-20241022',
+    'claude-3-haiku-20240307',
+    'claude-3-opus-20240229',
```

Comment:
```suggestion
    'gpt-4o',
    'gpt-4o-mini',
```

Reply:
OpenAI documents support for all models after November 2023, so it must be these and o1

Reply:
i haven't properly evaluated O1 yet, so i didn't put them there. I can put them there once i confirmed that it can work